mesonbuild/interpreter/interpreter.py

```diff
@@ -1894,6 +1896,15 @@ def func_install_man(self, node: mparser.BaseNode,
 
         return m
 
+    @FeatureNew('install_emptydir', '0.60.0')
+    @typed_kwargs(
+        'install_emptydir',
+        INSTALL_MODE_KW
+    )
+    def func_install_emptydir(self, node: mparser.BaseNode, args: T.Tuple[str], kwargs) -> None:
+        d = build.EmptyDir(args[0], kwargs['install_mode'], self.subproject)
+        self.build.emptydir.append(d)
```

Comment:
I think this should return d, right? Otherwise there really isn't any reason for build.EmptyDir to be Holdable, or for the EmptyDirHolder to exist

Reply:
Oops, yeah.